### PR TITLE
Support getting real content hash from hdfs when using getFileStatus

### PR DESCRIPTION
**Pull request by jja725**
### What changes are proposed in this pull request?

Support getting real content hash from hdfs

### Why are the changes needed?

for data integrity while moving data between object store & HDFS

### Does this PR introduce any user facing changes?
na

### CODE REVIEW

**Comment by tcrain** [comment]
Check here also please https://github.com/Alluxio/alluxio/blob/master/underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileOutputStream.java#L93

**Comment by jja725** [comment]
> Check here also please https://github.com/Alluxio/alluxio/blob/master/underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileOutputStream.java#L93

for this PR, it's specific to getFileStatus from ufs.

File: core/common/src/main/java/alluxio/conf/PropertyKey.java [hunk 1]
```diff
@@ -7546,6 +7546,12 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
           .setScope(Scope.SERVER)
           .build();
+  public static final PropertyKey HADOOP_CHECKSUM_COMBINE_MODE =
+      booleanBuilder(Name.HADOOP_CHECKSUM_COMBINE_MODE)
+          .setDescription("File Checksum combine mode.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.CLIENT)
```
Comment (beinan) [inline core/common/src/main/java/alluxio/conf/PropertyKey.java:7553]: is "client" the proper scope for the property?

**Review by beinan — APPROVED** [review]
lgtm except one clarification question.

File: core/common/src/main/java/alluxio/conf/PropertyKey.java (hunk 1 above)
Comment (jja725) [reply core/common/src/main/java/alluxio/conf/PropertyKey.java:7553]: at Hadoop side it's client property so I would keep the same behavior

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Comment by jja725** [comment]
alluxio-bot, cherry-pick this to dora, please

**Comment by jja725** [comment]
alluxio-bot, cherry-pick this to dora please

**Pull request merged**